src/components/AssetTypes/AddAssetTypesComponent.jsx

```diff
@@ -16,6 +16,7 @@ const AssetTypesComponent = props => (
   <Form onSubmit={props.handleSubmit}>
     <label htmlFor="asset-type">
       Asset Type
+
```

Comment:
I believe this change isn't important @HawiCaesar

Reply:
It was to trigger Hound to finish its check. I will remove it.